conductor_api/src/dpki_instance.rs

```diff
@@ -74,7 +80,8 @@ impl DpkiInstance for Holochain {
             DPKI_TRAIT_FN_IS_INITIALIZED,
             params,
         )?;
-        let result: bool = result.try_into()?;
-        Ok(result)
+        let result: IsInitializedResult = result.try_into()?;
```

Comment:
I don't thing you need to create a special Struct, you can just `try_into` a result:
```suggestion
        let result: Result<bool, HolochainError> = result.try_into()?;
```
This may cause confusions with different levels of errors because the issue is that the try_into itself may produce an err (hence the ?), and the `call` returns a json-encoded result as well, which may be an err.